_includes/releases/v23.1/v23.1.2.md

```diff
@@ -1,24 +1,209 @@
+- Fixed a bug where [Row-level TTL](../v23.1/row-level-ttl.html) jobs would incorrectly process a table that spanned multiple ranges in rare cases. This bug had been present since v22.2.0. You were affected by this bug if you saw the error message `"error decoding EncDatum of ...: did not find terminator ... in buffer ..."`. [#102881][#102881]
+- The `Retry.Max` field of the [`webhook_sink_config`](../v23.1/changefeed-sinks.html#webhook-sink) option for changefeeds was not correctly defined in the docs. The existing docs mention that it is "The maximum amount of time the sink will retry a single HTTP request to send a batch", but this is incorrect. It actually represents the maximum number of retries which will be attempted when sending a batch in an HTTP request fails. This is now fixed to correctly capture its behavior. Also, fixed a bug where the retry time would wait for 4 seconds, regardless of `Retry.Max`. With this change, the new maximum retry time is 30 seconds. The initial backoff will keep doubling every time a retry occurs until the maximum of 30 seconds is reached.  E.g. If `Retry.Max = 4` and the initial backoff is 10 seconds, at most 4 retries will be performed with the backoff times 10, 20, 30, 30 seconds respectively. [#102958][#102958]
+- Stopped using a `NULL` value for `pg_constraint.conparentid`. Now the value is hard-coded to `0`, since CockroachDB does not support constraints on [partitions](../v23.1/partitioning.html). [#103234][#103234]
+- Fixed a bug where [`COPY`](../v23.1/copy-from.html) in v23.1.0 and beta versions would incorrectly encode data with multiple column families. The data must be dropped and re-imported to be encoded correctly. [#103355][#103355]
+- Optimized over-head of [`pg_catalog.pg_description`](../v23.1/pg-catalog.html) and [`pg_catalog.pg_shdescription`](../v23.1/pg-catalog.html), which can lead to performance regression relative to v22.2 [#103331][#103331]
+- Timeseries [metric](../v23.1/metrics.html) counts will now show cumulative counts for a histogram rather than a windowed count. A `-sum` timeseries is also exported to keep track of the cumulative sum of all samples in the histogram. [#103444][#103444]
+- Fixed a bug where CockroachDB could produce incorrect results when evaluating queries with [`ORDER BY`](../v23.1/order-by.html) clause in rare circumstances. In particular, some rows could be duplicated if all of the following conditions were met: 
+    1. The query had a `LIMIT` clause.
+    2. The `SORT` operation had to spill to disk (meaning that `LIMIT` number of rows used up non-trivial amounts of memory, e.g. the rows were "wide").
+    3. The `ORDER BY` clause contained multiple columns **and** the ordering on the prefix of those columns was already provided by the index.  The bug has been present since at least v22.1. [#102790][#102790]
```

Comment:
Tried to shift the last bit out of the numbered item, but still under the list bullet. Hopefully this will render correctly if/when you commit the suggestion, not 100% sure the indentation is right; fiddly to do in the github review interface.
```suggestion
    1. The `ORDER BY` clause contained multiple columns **and** the ordering on the prefix of those columns was already provided by the index.
    
  The bug has been present since at least v22.1. [#102790][#102790]
```